packages/google_workspace/data_stream/gmail/fields/base-fields.yml

```
@@ -0,0 +1,20 @@
+- name: data_stream.type
+  type: constant_keyword
+  description: Data stream type.
+- name: data_stream.dataset
+  type: constant_keyword
+  description: Data stream dataset.
+- name: data_stream.namespace
+  type: constant_keyword
+  description: Data stream namespace.
+- name: event.module
+  type: constant_keyword
+  description: Event module.
+  value: google_workspace
+- name: event.dataset
+  type: constant_keyword
+  description: Event dataset.
+  value: google_workspace.gmail
+- name: '@timestamp'
+  type: date
+  description: Event timestamp.
```

Use `external: ecs` for all ECS fields to ensure consistency.

```suggestion
- name: data_stream.type
  external: ecs
- name: data_stream.dataset
  external: ecs
- name: data_stream.namespace
  external: ecs
- name: event.module
  type: constant_keyword
  external: ecs
  value: google_workspace
- name: event.dataset
  type: constant_keyword
  external: ecs
  value: google_workspace.gmail
- name: "@timestamp"
  external: ecs
```